.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
This is angular generated, we can remove but advantage of keeping this would be consisting way of development across contributors for example spaces, trimming wanted trail spaces etc.